.changeset/thick-pugs-appear.md

```diff
@@ -0,0 +1,5 @@
+---
+'@faustwp/cli': major
+---
+
+Removed Google Analytics endpoint and replaced with updated plugin API endpoint.
```

Comment:
Should we document in the changelog here why this is a breaking change? For example: "We removed <insert the function names we removed here>", so people can easily understand what is breaking without having to dig through PRs, or open support questions, or otherwise fret about what's going on?